examples/lighting-app/nrfconnect/main/AppTask.cpp

```diff
@@ -307,18 +309,47 @@ void AppTask::FunctionHandler(AppEvent * aEvent)
     }
 }
 
+void AppTask::StartThread()
+{
+    chip::DeviceLayer::Internal::DeviceNetworkInfo deviceNetworkInfo;
+    memset(&deviceNetworkInfo, 0, sizeof(deviceNetworkInfo));
+
+    const char* networkName = "OpenThreadDemo";
+    const uint64_t networkExtendedPANID = 0x2222222211111111;
```

Comment:
This assumes little-endian. If you decide not to use the Kconfig settings you may rewrite it as a byte-array.